src/RMGRunAction.cc

```diff
@@ -140,7 +145,7 @@ void RMGRunAction::BeginOfRunAction(const G4Run*) {
     // Warn user if persistency is disabled if there are detectors defined.
     auto level = manager->GetDetectorConstruction()->GetAllActiveOutputSchemes().empty()
                      ? RMGLog::summary
-                     : RMGLog::warning;
+                     : RMGLog::error;
```

Comment:
uhm are you sure? i don't have a strong option but i maybe feel this should be a warning

Reply:
you mean leaving it as-is? I am fine with that, that would also be my personal choice. In #326 it was just requested this will get an error....